pkg/kubelet/status/status_manager.go

```diff
@@ -994,15 +995,17 @@ func mergePodStatus(oldPodStatus, newPodStatus v1.PodStatus, couldHaveRunningCon
 
 	// Delay transitioning a pod to a terminal status unless the pod is actually terminal.
 	// The Kubelet should never transition a pod to terminal status that could have running
-	// containers and thus actively be leveraging exclusive resources. Note that resources
+	// containers or unprepared dynamic resources and thus actively be leveraging exclusive
+	// resources.
+	// Note that resources
```

Comment:
I wouldn't get so specific here, otherwise we have to keep this in sync with the other comments and the implementation of `ShouldDelayTransitioningPodToTerminal()`

Reply:
done